### PR TITLE
Autopilot > LoS: UI issues

### LINKED ISSUE #3846: Autopilot > LoS: UI issues
**Description**: The list of speakers has UI problems on small screens (e.g. iPhone SE) and requires optimization.

1. The space to the left of the individual elements is not used. (See image.)

image

2. Last speakers: If people are on this list, an additional scrollbar is displayed at the bottom of the screen.

Example image:
image

**Reproduction**:
1. open a meeting
2. open dev-tools and use scaling for IPhone SE
3. open agenda > create item > project
4. open autopilot > list of speakers > add a Speaker (here you 1.)
5. start und stop speaker (now you see the scrollbar for 2. issue)

**What should happen:**
1. Arrange elements more to the left on small screens in order to utilize the available space. Maybe the buttons 'Add me' and 'Point of order' could than also switch in one line.
2. Make better use of the available space so that no scrollbar or a scrollbar only appears on very small screens. Possibly the first problem causes the second problem.

### PR DESCRIPTION
**Pull request by vkrasnovyd**
Resolve #3846 

1. Fixed: the space to the left of the individual elements is not used in multiple sections:
1.1 In Autopilot: list of speakers, list of previuos speakers.
1.2 In polls > Poll detail page > Candidates list.

2. The horizontal scrollbar appeared because the components of Autopilot page were put inside the container 'content-container' that added margins on the sides. Removed this container to let the content fully adjust to the screen size.
The buttons 'Add me' and 'Point of order' switched in one line indeed.
Note: used poll detail page as reference where 'content-container' wasn't used.

### CODE REVIEW

**Review by Elblinator — CHANGES_REQUESTED** [review]
The placment of the structure level card changed, please change it back:
If the screen is 1050+px wide then the structure level card should be right from the other contents on the autopilot
(note that I configureted my autopilot to only show the LoS and the speaking times card)
**wanted appearance:**
Screenshot_20240802_105711

**currently:**
Screenshot_20240802_105721
If the other things would be visible in my autopilot then the speaking times card would be below all of them


**How to activate to structure level card:**
1. navigate to settings > List of speakers > third to last point 
2. activate the "Default speaking time contingent for parliamentary groups(structure levels) in seconds"
3. project an object with a LoS onto the Autopilot (e.g. a motion)
4. navigate to participants > three-dot-menu > structure levels
5. create a structure level
6. assign a participant a structure level
7. navigate to the autopilot
8. put someone with a structure level onto the LoS


The rest of the changed appearance is fine :)

**Comment by vkrasnovyd** [comment]
It turned out that the removed container was needed to keep timer in the separate column right on big screen. Reverted this change.

The 'display: flex' property inside this container however didn't let the blocks of speakers info properly adjust to small screens. Changed it to 'display: block' for phones only.

File: client/src/styles.scss [hunk 1]
```diff
@@ -289,10 +289,9 @@ button.mat-mdc-menu-item.selected {
     font-size: 16px;
 }
 
-.queue {
-    .mat-expansion-panel-body {
-        padding: 0 !important;
-    }
+.mat-expansion-panel-body {
+    padding: 0 !important;
+    margin-bottom: 16px;
```
Comment (bastianjoel) [inline client/src/styles.scss:294]: I am not sure if this change was intended. If not please revert.  

```suggestion
.queue {
    .mat-expansion-panel-body {
        padding: 0 !important;
    }
```

**Review by bastianjoel — APPROVED** [review]

File: client/src/styles.scss (hunk 1 above)
Comment (bastianjoel) [reply client/src/styles.scss]: I also meant the removal of the wrapping `.queue`.

**Comment by vkrasnovyd** [comment]
Added one more change for better space usage: removed side margins for 'add-self-button' (for small screens only). Now buttons are aligned with the list of speakers on the left side.

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]

**Pull request merged**